app/celery/broadcast_message_tasks.py

```diff
@@ -45,4 +49,6 @@ def send_broadcast_event(broadcast_event_id):
             description=broadcast_event.transmitted_content['body'],
             areas=areas,
```

Comment:
I'm not sure we need the areas for a cancellation message

Reply:
`git grep 'unsubstantiated reckon'` 😆

Reply:
We can tidy these up when we implement cancels and updates. Don't let it block merging.